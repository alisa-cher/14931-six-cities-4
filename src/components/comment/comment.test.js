import React from 'react';
import renderer from 'react-test-renderer';
import Comment from '../comment/comment.jsx';

const comment = {
  user: {
    name: `Emely`,
    avatarUrl: `https://htmlacademy-react-3.appspot.com/six-cities/static/avatar/8.jpg`},
  rating: 4,
  comment: `Beautiful space, fantastic location and atmosphere, really a wonderful place to spend a few days. Will be back.`,
  date: `2020-06-17T10:54:44.806Z`
};

it(`Comment renders correctly`, () => {
  const tree = renderer
    .create(
        <Comment
          comment={comment}
          tag={`li`}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
