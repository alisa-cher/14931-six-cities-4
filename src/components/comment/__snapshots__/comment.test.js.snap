// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment renders correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="https://htmlacademy-react-3.appspot.com/six-cities/static/avatar/8.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Emely
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "80%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Beautiful space, fantastic location and atmosphere, really a wonderful place to spend a few days. Will be back.
    </p>
    <time
      className="reviews__time"
      dateTime="2020-6-17"
    >
      June 
      2020
    </time>
  </div>
</li>
`;
